modules/op-executing-pipeline-run-and-task-run-with-custom-scc-and-service-account.adoc

```diff
@@ -0,0 +1,91 @@
+[id=op-executing-pipeline-run-and-task-run-with-custom-scc-and-service-account_{context}]
+= Executing pipeline run and task run with custom SCC and service account
+
+When using the `pipelines-scc` security context contraint (SCC) associated with the default `pipelines` service account (SA), the pipeline run and task run pods may face timeouts. This happens because in the default `pipelines-scc` SCC, the `fsGroup.type` parameter is set to `MustRunAs`.
```

Comment:
We should point to the reason why it can timeout (the access.redhat.com link or some bug report somewhere)

Reply:
Just a personal preference, but I wouldn't use `(SA)` here. I don't think service account is really a long enough term to need to be abbreviated and would be less confusing if it was just spelled out each time. Up to you though :)